andela/wger-space
# #166185778 Enable Social Login

**What does this PR do?**

- Enables users to use social login via facebook and google

**Description of Task to be completed?**

- add social login links to login page
- add social login library and configure it to handle social login functionality
- obtain API keys for all the social login platforms

**How should this be manually tested**

- clone the repository and setup the project using the description in the readme file
- checkout to the branch 'ft-social-login-166185778'
- Install social-django using pip install -r requirements.txt
- load the login page at 'http://localhost:8000/en/user/login'
- You will be able to see the social login links that will enable you to login

**what are the relevant pivotal tracker stories**

[#166185778](https://www.pivotaltracker.com/story/show/166185778)

**Screenshots**

<img width="1201" alt="Screen Shot 2019-06-02 at 19 38 47" src="https://user-images.githubusercontent.com/40593659/58764301-22e76e80-856e-11e9-844d-7a642fceba8b.png">
